### ISSUE TITLE
Melfest, after accidentally double clicking the active moment you can not open it

### ISSUE THREAD

**Issue by fredrik-jarnbrost** [stagecast/bug-report#148]
**Describe the bug**
If you accidentally double click the active moment, sometimes you can not open it

**To Reproduce**
1. Launch any moment from the webapp
2. Open the app on device
3. Close the moment
4. From the active moments/feed view, double click the active moment
5. It does not open, and can not be opened with subsequent clicks

**Expected behavior**
Accidentally double clicking should not break the button

**Smartphone (please complete the following information):**
 - Device: iPhone 6S
 - OS: iOS 12.1.2
 - Version: Testflight StagecastSDK 1.0 (5.0)

**Comment by peffis** [comment]
I'm having problems to reproduce this. No matter how much I try to double tap it just opens the moment. Is this easy to reproduce on your phone? I will look into the code and see if there is something that can be improved when it comes to robustness of opening though but you would need to help me test it later

**Comment by peffis** [comment]
I will make some "improvement" to this, but I will keep this open to see if you will be able to reproduce it when build 0.0.10 is out.

**Comment by fredrik-jarnbrost** [comment]
I can still reproduce on 0.0.10, here's a clip how I do:
https://drive.google.com/open?id=1IZggHeTIwKOp4Fa8jk7vnyDBSB9FXU7x

**Comment by fredrik-jarnbrost** [comment]
Still reproducible on 0.0.12

**Comment by fredrik-jarnbrost** [comment]
This might be interesting with regards to this bug: https://github.com/react-navigation/react-navigation/issues/271

**Comment by peffis** [comment]
Yes, the debounce approach mentioned there was the last I tried. I was surprised that that did not solve it either. 

/S
